### PR TITLE
Some small fixes and style

### PR DESCRIPTION
**Pull request by dlemures**
Some small fixes and style

### CODE REVIEW

File: toothpick-runtime/src/main/java/toothpick/ToothPick.java [hunk 1]
```diff
@@ -36,41 +36,46 @@ public static Scope openScopes(Object... names) {
...
   public static void reset() {
-    MAP_KEY_TO_INJECTOR.clear();
+    MAP_KEY_TO_SCOPE.clear();
   }
 
   public static void inject(Object obj, Scope scope) {
     injector.inject(obj, scope);
   }
+
+  private static void removeScopeAndChildrenFromMap(Scope scope) {
```
Comment (stephanenicolas) [inline toothpick-runtime/src/main/java/toothpick/ToothPick.java]: you should add a comment to say this method is not synchronized.

File: toothpick-runtime/src/main/java/toothpick/ToothPick.java [hunk 2]
```diff
@@ -36,41 +36,46 @@ public static Scope openScopes(Object... names) {
...
+      if (scope == null) {
+        scope = new ScopeImpl(name);
+        MAP_KEY_TO_SCOPE.put(name, scope);
       }
+      return scope;
     }
-    return scope;
   }
 
-  public static void closeScope(Object key) {
```
Comment (stephanenicolas) [inline toothpick-runtime/src/main/java/toothpick/ToothPick.java]: ok

File: toothpick-runtime/src/main/java/toothpick/ToothPick.java (hunk 1 above)
Comment (dlemures) [inline toothpick-runtime/src/main/java/toothpick/ToothPick.java]: added

**Comment by dlemures** [comment]
ConcurrentHashMap not needed, we need to synchronize where the map is used anyway.

There was the option to keep the ConcurrentHashMap to allow multiple "closeScope" at the same time for some special cases. However, it is too complicated and asymmetric (open vs close), so the improvement it is not worth.

**Pull request merged**